src/VolumeDrawable.ts

```diff
@@ -450,17 +470,20 @@ export default class VolumeDrawable {
   }
 
   onChannelLoaded(batch: number[]): void {
-    this.meshVolume.onChannelData(batch);
```

Comment:
What did this line do before?

Reply:
See the changes to MeshVolume.ts, where I removed this function. It was doing something similar to the block of added code below - updating an isosurface on data load - but was written with an older version of the app in mind, where a volume receives a complete set of loaded data once and never changes again. I decided to move those updates up to this function rather than rework the existing method because `MeshVolume` only has access to the enabled/disabled state of an isosurface based on whether or not it has a three.js mesh object for it, which doesn't perfectly correspond to whether the isosurface is actually visible.